setup/build.cmd

```diff
@@ -4,11 +4,11 @@
 @REM Requires python 2.7, likely from conda env.
 SET PYTHON=python
 SET IPY=%PROGRAMFILES%\IronPython 2.7\ipy.exe
-SET HB=C:\Users\Christoph\Documents\GitHub\honey-badger\honey-badger.py
+SET HB=C:\Users\chwaibel\Documents\GitHub\honey-badger\honey-badger.py
 SET MSBUILD=%PROGRAMFILES(x86)%\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\MSBuild.exe
 SET MAKENSIS=%PROGRAMFILES(X86)%\NSIS\makensis.exe
 
-SET HIVEWIKIPY=C:\Users\Christoph\Documents\GitHub\hive.wiki\hbwiki.py
+SET HIVEWIKIPY=C:\Users\chwaibel\Documents\GitHub\hive.wiki\hbwiki.py
```

Comment:
you can probably use environment variables here: `%USERPROFILE%` refers to your home directory. BUT: see [here on stackoverflow](https://stackoverflow.com/questions/17063947/get-current-batchfile-directory) for a more interesting idea: `%~dp0` refers to the drive and the directory of the batch file itself - this is useful, when you're not the only developer...

Reply:
ah cool thanks for the tip!